lib/manifest.ps1

```diff
@@ -102,7 +118,6 @@ function generate_user_manifest($app, $bucket, $version) {
 function url($manifest, $arch) { arch_specific 'url' $manifest $arch }
 function installer($manifest, $arch) { arch_specific 'installer' $manifest $arch }
 function uninstaller($manifest, $arch) { arch_specific 'uninstaller' $manifest $arch }
-function msi($manifest, $arch) { arch_specific 'msi' $manifest $arch }
```

Comment:
Why do you deprecate this function inside this PR??

Instead of atomic PR for one thing you are deprecating some functionality. Why?

Reply:
Refactored `dl_urls` and `run_installer`, and deprecated `msi` is fully deleted.